pkg/vault/ledger/ledger/devices.go

```diff
@@ -20,43 +20,55 @@ type LedgerDeviceInfo struct {
 	LegacyUSBProductID uint16
 	USBOnly            bool
 	MemorySize         int
-	BlockSize          int
-	BluetoothSpec      []*BluetoothSpec
 }
 
-var ledgerDevices = []*LedgerDeviceInfo{
-	&LedgerDeviceInfo{
+var (
+	ledgerBlue = LedgerDeviceInfo{
 		ID:                 "blue",
 		ProductName:        "Ledger Blue",
 		ProductIDMM:        0x00,
 		LegacyUSBProductID: 0x0000,
 		USBOnly:            true,
 		MemorySize:         480 * 1024,
-		BlockSize:          4 * 1024,
```

Comment:
was removing  BlockSize from the devices' infos an intentional change?  seems unrelated to the PR summary of adding tcp apdu transport

Reply:
was sanity tested with Nano S, so, probably fine.

Reply:
I think it would be better to only make changes required to add speculos support, that way we won't be introducing untested changes.

Reply:
this has been tested on nanoS, nanoSP, and nanoX